src/formatter.rs

```diff
@@ -1,9 +1,11 @@
 //! Functions related to formatting and printing lines from a `Tokenizer`.
 
-use crate::{extensions::FindFrom, types::LineType};
+use std::result::Result as StdResult;
```

Comment:
Actually, we don't need that. The type alias `anyhow::Result` still has `E` as a generic parameter, `anyhow::Error` is just the default

Reply:
Oh wow, TIL.